Cargo.toml

```diff
@@ -44,12 +35,15 @@ socket2 = "0.5.7"
 yaml-rust = "0.4.5"
 x86 = "0.52.0"
 
-# Local crates.
-nettest = { path = "nettest" }
-dpdk-rs = { path = "dpdk-rs", optional = true }
+# Demikernel crates (published on crates.io).
+# The demikernel-dpdk-rs crate is optional because it is only used for DPDK support.
+demikernel-dpdk-rs = { version = "0.0.1", optional = true }
```

Comment:
is there a reason to prefix with demikernel for crates.io? Is there another crate already named dpdk-rs? Also if we are going to rename the crate, the rs seems kind of redundant.

Reply:
* Crate names: I added the Demikernel prefix so that all Demikernel crates are easily searchable. I kept the name in code to match the crate names. We can change the names to remove -rs suffix.

Reply:
Ok, sounds good!